openshift-ci/build-root/OWNERS

```diff
@@ -0,0 +1,19 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- sbose78
+- wtam2018
+- bigkevmcd
+- chetan-rns
+
+reviewers:
+- sbose78
+- wtam2018
+- bigkevmcd
+- chetan-rns
+- mitchharpur
+- keithchong
+- tetchel
+- amitkrout
+- jaideepr97
+- gaganhegde
```

Comment:
Place this OWNER file into the root directory. Folks listed in the owner file will get the lgtm and approve privilege from the root level.

Reply:
Done with b09222c8ba25e1c5d1cdc88a3b890335f070662a